x-pack/plugins/security/server/routes/views/logged_out.ts

```diff
@@ -39,10 +38,7 @@ export function defineLoggedOutRoutes({
         });
       }
 
-      return response.ok({
-        body: await context.core.rendering.render({ includeUserSettings: false }),
-        headers: { 'content-security-policy': csp.header },
-      });
+      return response.renderAnonymousCoreApp();
```

Comment:
Should we use `httpResources.registerAnonymousCoreApp` instead?

Reply:
No, there is some logic in a route handler. https://github.com/elastic/kibana/pull/61797/files/b9f309947e63741457f567a908f2af3c50e42e2b#diff-f2a8e41fcb33260f01bdca9a2ed1854eR33-R39